VAMobile/documentation/docs/Engineering/DevOps/Creating a Firebase Project.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Creating a Firebase Project
+postion: 3
+---
+
+## Introduction
+
+This document will go through the steps of creating a firebase project.
+
+:::info
+You will need access to the `va-mobile-app` in Firebase before you can complete this task. Please reach out to your manager if you do not have access.
```

Comment:
I think this would be better on the normal firebase page: https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Operations/App%20metrics/Firebase